src/main/java/calculator/domain/Addition.java

```diff
@@ -0,0 +1,7 @@
+package calculator.domain;
+
+public final class Addition extends Operation {
+	public Digit operate(final Digit digit1, final Digit digit2) {
+		return new Digit(digit1.digit() + digit2.digit());
```

Comment:
```suggestion
		return digit1.add(digit2);
```
위와 같이 값을 꺼내지 말고 메시지를 보내 구현하는 것은 어떨까?